Specs/DateTime/French/DateTimeModelCalendarMode

```diff
@@ -0,0 +1,951 @@
+[
```

Comment:
All spec files must have names ending in json. Can you please correct this one and the others?